go/consensus/tendermint/apps/keymanager/keymanager.go

```diff
@@ -120,15 +120,15 @@ func (app *keymanagerApplication) onEpochChange(ctx *tmapi.Context, epoch beacon
 			acctAddr := rt.StakingAddress()
 			if acctAddr == nil {
 				// This should never happen.
-				ctx.Logger().Error("unknown runtime governance model",
+				ctx.Logger().Debug("unknown runtime governance model",
```

Comment:
This results in a consensus halt, so it should stay as error.

Reply:
Thank you. Reverted to `Error`; same for the identical log message in `roothash.go` (as noted by you below) and in `go/consensus/tendermint/apps/keymanager/keymanager.go` (which you didn't call out; LMK if that was intentional).